index.html

```diff
@@ -1,204 +1,284 @@
         <button class="view-project resume">Get my resume</button>
         <section class="main-skills">
-
           <div class="languages">
             <h4>languages</h4>
-            <img class="arrow" width="14px" height="8px" src="./imgs/arrow-down.png" alt="arrow">
+            <img
+              class="arrow"
+              width="14px"
+              height="8px"
+              src="./imgs/arrow-down.png"
+              alt="arrow"
+            />
           </div>
           <ul class="lang-group">
             <li class="lang">
-              <img src="./imgs/javascript.png" alt="javaScript">
+              <img src="./imgs/javascript.png" alt="javaScript" />
               <h6>javaScript</h6>
             </li>
             <li class="lang">
-              <img src="./imgs/html.png" alt="html">
+              <img src="./imgs/html.png" alt="html" />
               <h6>HTML</h6>
             </li>
             <li class="lang">
-              <img src="./imgs/css.png" alt="css">
+              <img src="./imgs/css.png" alt="css" />
               <h6>CSS</h6>
             </li>
           </ul>
-          </div>
+
           <ul>
             <li class="framewroks">
               <h4>frameworks</h4>
-              <img class="arrow" src="./imgs/arrow-left.png" alt="arrow">
+              <img class="arrow" src="./imgs/arrow-left.png" alt="arrow" />
             </li>
             <li class="some-skill">
               <h4>skills</h4>
-              <img class="arrow" src="./imgs/arrow-left.png" alt="arrow">
+              <img class="arrow" src="./imgs/arrow-left.png" alt="arrow" />
             </li>
           </ul>
         </section>
-    </section>
-  </main>
-</body>
-
-</html>
\ No newline at end of file
+      </section>
+      <section class="contact-form">
+        <div class="contact-group">
+          <h2 class="contact-title">Contact me</h2>
+          <p class="contact-desc">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+        <form action="https://formspree.io/f/xqkjqjeq" method="POST">
+          <label>
+            <input
+              type="text"
+              placeholder="Write your name"
+              required
+              maxlength="30"
+              name="name"
+            />
+          </label>
+          <label>
+            <input
+              type="email"
+              placeholder="Write your email here"
+              required
+              name="email"
+            />
+          </label>
+          <label>
+            <textarea
+              placeholder="Write your message here"
+              rows="10"
+              required
+              name="message"
+              maxlength="500"
+            ></textarea>
+          </label>
+          <button class="submit" type="submit">Get It Touch</button>
+        </form>
```

Comment:
Great job team 🤝

- [ ] Kindly add the dark line at the bottom of your design just like it appears on the Figma template.
- Your design
![your design](https://user-images.githubusercontent.com/63917725/192567047-94c84fad-97e1-4715-ac67-e32303ba5701.PNG)

- Figma design
![Figma line](https://user-images.githubusercontent.com/63917725/192567274-f3e22584-542d-4083-9c91-559082e6da9b.PNG)